graphql_compiler/query_pagination/query_splitter.py

```diff
@@ -23,11 +25,13 @@ def split_into_page_query_and_remainder_query(
         num_pages: int, number of pages to split the query into.
 
     Returns:
-        tuple of (ASTWithParameters namedtuple, ASTWithParameters namedtuple), describing two
-        queries: the first query when executed will return roughly a page of results of the original
-        query; the second query will return the rest of the results of the original query. The union
-        of the two queries' result data is equivalent to the given query and parameter's result
-        data. There are no guarantees on the order of the result rows for the two generated queries.
+        tuple containing three elements:
+            - ASTWithParameters namedtuple, describing a query expected to return roughly a page
```

Comment:
nit: ASTWithParameters is not a namedtuple. Could you please fix this everywhere else?